app/locales/en.json

```
@@ -183,6 +183,10 @@
     "epidemiologic_report_tab": "Epidemiological report",
     "mental_health_advice_tab": "Mental health tips"
   },
+  "maps": {
+    "error-header": "We are sorry",
```

Please use underscore to continue the standard in the project

```suggestion
    "error_header": "We are sorry",
```